Code/CSharpOOP2/Homework.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP2
+{
+    public class Table
+    {
+        public string Name;
+        public int Quantity;
+        public double Price = 99.99;// Default value for the Price field (Task_13)
+        public string Brand;
+        public bool IsAvailable;
+        public void PrintAvailability()
+        {
+            if (Quantity > 0)
+            {
+                Console.WriteLine("Table {0} is avalible.", Name);
+            }
+            else
+            {
+                Console.WriteLine("Product {0} is temporarily out of stock.", Name);
+            }
+        }
+        //Class constructor Task_6
+        public Table(string name, int quantity, double price, string brand, bool isAvailable)
```

Comment:
just a minor comment: constructors should be above fields and fields should be above methods according to style code

Reply:
done